cli/src/cli.rs

```diff
@@ -847,6 +864,86 @@ pub fn parse_command(
     Ok(response)
 }
 
+/// Request information about node's stake
+/// If fail to get requested information, return error
+/// Otherwise return stake of the node
+/// along with total activated stake of the network
+fn find_node_activated_stake(
```

Comment:
Don't we move to the new the implementation of qos with 1 stream per connection? So with this we don't need to calculate number of streams per connection and this code is no longer needed?

Reply:
But if there is no sense to open more than one streams, why should we move this code which is used to compute number of streams per connection based on the stake of validator identity?
This number is used to send concurrently txs, one tx per stream, using a vec of futures and `join_all`. It sounds unnecessary if 1 concurrent stream is enough